en/python_installation/instructions.md

```diff
@@ -4,6 +4,7 @@
 
 Django is written in Python. We need Python to do anything in Django. Let's start by installing it! We want you to install the latest version of Python 3, so if you have any earlier version, you will need to upgrade it. If you already have version 3.4 or higher you should be fine.
 
+If you have Anaconda installed on your computer, please install normal Python as follows.
```

Comment:
Those who don't have Anaconda installed, should of course install normal Python, too. Thus I'd suggest to phrase it with "even when":

```suggestion
Please install normal Python as follows, even when you have Anaconda installed on your computer.
```

Reply:
@das-g Thank you for fixing. I also think that is better.